DATA/SHIPS/shiparch.ini

```diff
@@ -5628,7 +5629,7 @@ DA_archetype = ships\bretonia\br_destroyer\br_destroyer.cmp
 material_library = ships\bretonia\br_capships.mat
 material_library = fx\envmapbasic.mat
 envmap_material = envmapbasic
-linear_drag = 10000
+linear_drag = 24000

```

Comment:
The increased linear drag here and at the battleship reduce the feeling of them being heavy. They break too fast, taking away the illusion of inertia.